code/src/Components/RadioButtons.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+const gamesList = [
+    'Beat Saber',
+    'SPARC',
+    'Until you Fall',
+    'Pistol Whip',
+    'No Man\'s Sky',
+    'Half-Life: Alyx'
+]
+export const RadioButtons = ({game, setGamesList}) => {
```

Comment:
For some reason, when tab-ing through your survey it took many tabs to reach the radio buttons and their options. I ended up on the Next-button first and then in a lot of menus in my browser before I got to the radio buttons while only tabing forward. Unfortunately, I don't know if there is anything you can do about it :/

Reply:
I did have an issue with the `focus` and solved it by having `autoFocus` but don't know why doesn't work in here to be honest :\